peg_src/jsdoctype.pegjs

```diff
@@ -812,12 +794,12 @@ ArrowTypeExpr = newModifier:"new"? _ paramsPart:ArrowTypeExprParamsList _ "=>" _
                    };
 }
 
-ArrowTypeExprParamsList = "(" _ params:ArrowTypeExprParams _ ")" {
-                            return params;
-                          }
-                        / "(" _ ")" {
-                            return [];
-                          }
+ArrowTypeExprParamsList = "(" _ ")" {
```

Comment:
This was just for coverage as the code for checking the empty params case wouldn't be found otherwise. This is because all of the components of `ArrowTypeExprParams` are optional, so it can match the empty array example as well. If you prefer, I can just remove the empty array code entirely.

Reply:
It's a *little* confusing, but overall I think it's fine.

Reply:
Ok, let me know if there is anything else to do.

Reply:
No, I think it's good to go.